Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -291,6 +291,22 @@ public static void DismissAlert(this UITestContext context)
         context.Driver.SwitchTo().DefaultContent();
     }
 
+    /// <summary>
+    /// Clicks on the first matching element, switches control to JS alert/prompt box, if <paramref name="inputText"/>
+    /// is not <see langword="null"/> enters it as keystrokes, accepts the alert/prompt box, and switches control back
+    /// to main document or first frame.
+    /// </summary>
+    public static void ClickAndAcceptPrompt(this UITestContext context, By by, string inputText = null)
+    {
+        // It uses FindElement because using ClickReliablyOnAsync would throw a "Unexpected Alert Open" exception.
```

Comment:
Thanks for adding this hint.

```suggestion
        // Using FindElement() here because ClickReliablyOnAsync() would throw an "Unexpected Alert Open" exception.
```